build.gradle

```diff
@@ -96,6 +99,7 @@ subprojects {
     }
 
     signing {
+        useInMemoryPgpKeys(System.getenv('PGP_PRIVATE_KEY'), System.getenv('PGP_KEY_PASSPHRASE'))
```

Comment:
How are these secret environment variables set and managed? I found some documentation for how secret values can be passed to GitHub actions using environment variables, but wasn't able to find any in the repository: https://docs.github.com/en/actions/learn-github-actions/variables#creating-configuration-variables-for-a-repository

Maybe I don't have permissions to manage the secrets in the repo?